web/src/beta/lib/core/StoryPanel/Block/builtin/Camera/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { useReactiveVar } from "@apollo/client";
+import { useCallback, useEffect, useMemo } from "react";
+
+import { useVisualizer } from "@reearth/beta/lib/core/Visualizer/context";
+import { ValueTypes } from "@reearth/beta/utils/value";
+import { currentCameraVar } from "@reearth/services/state";
+
+import { getFieldValue } from "../../../utils";
+import { CommonProps as BlockProps } from "../../types";
+import usePropertyValueUpdate from "../common/usePropertyValueUpdate";
+import BlockWrapper from "../common/Wrapper";
+
+import CameraEditor, { Props as EditorProps } from "./Editor";
+
+export type Props = BlockProps;
+
+const CameraBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
+  const {
+    handlePropertyValueUpdate,
+    handleAddPropertyItem,
+    handleRemovePropertyItem,
+    handleMovePropertyItem,
+  } = usePropertyValueUpdate();
+
+  const visualizer = useVisualizer();
+
+  const currentCamera = useReactiveVar(currentCameraVar);
+  const handleFlyTo = useMemo(() => visualizer.current?.engine.flyTo, [visualizer]);
+
+  const items = useMemo(
+    () => getFieldValue(block?.property?.items ?? [], "") as EditorProps["items"],
+    [block?.property?.items],
+  );
+
+  const handleAddItem = useCallback(() => {
```

Comment:
handleNounVerb. So here it should be `handleItemAdd`
 Please update all callbacks where appropriate 🙏🏻 
Reason for this is readability
```
handleItemAdd,
handleItemRemove,
handleItemUpdate,
```
is much more readable than
```
handleAddItem,
handleRemoveItem,
etc
```

Reply:
Apologies, I missed that even though I knew about it. Fixed it now!